### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by marmoure**
This PR depends on #7 
Bump the version of a few dependencies

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>xml-apis</groupId>
             <artifactId>xml-apis</artifactId>
-            <version>1.4.01</version>
+            <version>2.0.2</version>
```
Comment (adamretter) [inline pom.xml]: As this is `provided` it comes from eXist-db, the version must match that used in eXist-db. Can you fix please?

**Review by adamretter — CHANGES_REQUESTED** [review]
Small fix needed

File: pom.xml (hunk 1 above)
Comment (marmoure) [reply pom.xml]: @adamretter can you please point out where it's declared in exist-db
i skimmed through https://github.com/eXist-db/exist/tree/develop-6.x.x `pom.xml` 
and it was declared like this
```xml
        <dependency>
            <groupId>xml-apis</groupId>
            <artifactId>xml-apis</artifactId>
        </dependency>
```
without any version number!!
does that mean it using the latest?

Comment (adamretter) [reply pom.xml]: I think a quick `grep` would have located this for you - https://github.com/eXist-db/exist/blob/develop-6.x.x/exist-parent/pom.xml#L448

**Pull request merged**